invenio_openaire/loaders.py

```diff
@@ -261,7 +280,7 @@ def __init__(self, source=None, setspec=None, **kwargs):
         self.setspec = setspec or \
             current_app.config['OPENAIRE_OAIPMH_DEFAULT_SET'],
 
-    def iter_grants(self):
+    def iter_grants(self, as_json=True):
```

Comment:
What about creating `iter_raw_grants` instead and changing implementation as follows?

```
def iter_grants(self):
    for grant in self.iter_raw_grants():
        yield self.grantxml2json(grant)
```